### ISSUE TITLE
Floating Point Exception with Conv3d 

### ISSUE THREAD

**Issue by slundqui** [tensorflow/tensorflow#4521]
I am getting a floating point exception with the following example:

``` python
import numpy as np
import tensorflow as tf

inputShape = (16, 3, 64, 256, 6)
weightShape = (2, 16, 32, 6, 1028)

np_in = np.random.rand(*inputShape).astype(np.float32)
np_w = np.random.rand(*weightShape).astype(np.float32)

tf_in = tf.Variable(np_in)
tf_w = tf.Variable(np_w)

tf_out = tf.nn.conv3d(tf_in, tf_w, [1, 1, 1, 1, 1], padding="SAME")

init = tf.initialize_all_variables()
sess = tf.Session()
sess.run(init)

np_out = sess.run(tf_out)
```

I'm currently using Cuda 8.0.27 and CUDNN 5.1.5, running on Ubuntu 16.04.1 LTS
TensorFlow built from source, commit a0d929df36ca7d6f72184371f6c3d6d877dded3e
Bazel version 0.3.1

The code does not error out if you change the kernel height (currently 16) to other numbers (I've tried 14, 15, 17, and 18).

Thanks in advanced.

**Comment by jmchen-g** [comment]
what was the error message?

**Comment by slundqui** [comment]
I get the following error message:
Floating point exception (core dumped)

**Comment by aselle** [comment]
It seems to work on CPU only for me, does it work on CPU for you?

**Comment by slundqui** [comment]
This does seem to be a GPU specific bug, as the code works fine on the CPU.

**Comment by zheng-xq** [comment]
Passing this to Michal.

**Comment by gmazzamuto** [comment]
Hi all,
I have a very similar problem. This MWE fails with a floating point exception only when `w_size = 16`:
```python
import tensorflow as tf

w_size = 16

at = tf.random_normal((1, 128, 128, 5, 1))
bt = tf.random_normal((w_size, w_size, 5, 1, 1))

with tf.device('/gpu:0'):
    ct = tf.nn.convolution(at, bt, "SAME", strides=[w_size, w_size, 5])
    
with tf.Session() as sess:
    sess.run(ct)
```
Tested with TensorFlow 0.12.0, CUDA 8.0.44, cuDNN 5.1 on the following systems:
* Ubuntu 16.04.1 LTS, Tesla K40c, driver version 375.20
* Ubuntu 14.04.5 LTS, GeForce GTX 980, driver version 367.57

**Comment by mrajchl** [comment]
I can independently confirm this bug. Tried this with Tensorflow 1.0, [CUDA 8.0.44, 8.0.61], cuDNN 5.1, Ubuntu 16.04 LTS, [Tesla K80, GTX 1080]. This even crashes at w_size = 8 in some cases. Are there any updates?

**Comment by yzhwang** [comment]
@mrajchl @slundqui Could you test it on tensorflow 1.1 and see if this bug still exists?
Also, @mjanusz , any update on this?

**Comment by gmazzamuto** [comment]
Seems to be working fine with TensorFlow 1.2.0!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yzhwang** [comment]
Verified with TensorFlow 1.4.1 and it has been fixed.
